android/docker/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:24.04@sha256:2e863c44b718727c860746568e1d54afd13b2fa71b160f5cd9058fc436217b30
+
+RUN apt-get update  \
+    && apt-get install -y git openjdk-17-jdk unzip wget gcc g++  \
+    && rm -rf /var/lib/apt/lists/*
+
+ENV ANDROID_COMMAND_LINE_TOOLS_FILENAME commandlinetools-linux-10406996_latest.zip
+ENV ANDROID_API_LEVELS                  android-34
+ENV ANDROID_BUILD_TOOLS_VERSION         34.0.0
```

Comment:
1. These variables and their values should be taken from the repo, to avoid copy-paste and sync issues.
2. Can Dockerfile be in the root folder? Why is it in the docker subfolder?